### PR TITLE
D8CORE-3042 Adjusted css for youtube videos in paragraph types

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Fixed the iframe containers with videos in cards with multiple per row.

# Need Review By (Date)
- 11/25

# Urgency
- high

# Steps to Test
1. checkout this branch and clear browser cache
1. create a basic page
1. put 3 rows with 1, 2, & 3 items. Each row should contain a card/media w/ caption
1. populate the items with a youtube video
1. save and ensure the youtube videos don't look wrong.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: lib/scss/components/card.component.scss [hunk 1]
```diff
@@ -4,17 +4,13 @@
 @import 'decanter/core/src/scss/components/card/index';
 
 .su-card {
-  .media {
-    &.video {
-      position: relative;
-      padding-top: 56.25%;
+  > div {
+    @include aspect-ratio(16, 9, '> div');
```
Comment (sherakama) [inline lib/scss/components/card.component.scss:8]: ❤️

**Comment by pookmish** [comment]
I think i've got one more fix similar to these for wysiwyg fields. hang tight

**Comment by pookmish** [comment]
g2g now with the wysiwyg style

**Comment by pookmish** [comment]
Not ready yet. found some issues with it

**Comment by pookmish** [comment]
Ok nevermind, i think i had something wrong on my system. this looks good to me. I've tested with backstop js and everything is identical except for some spacing changes which appear to be desired due to the aspect ratio changing.

**Comment by pookmish** [comment]
Heres a couple screenshots. the left is the `8.x-1.x` branch. the right is this branch.
pjimage (1)

pjimage

**Comment by jenbreese** [comment]
Thanks for the screenshots. I was setting up the test incorrectly

**Comment by jenbreese** [comment]
@pookmish I'm not getting the same result as the screenshots. Any ideas on why? Could I be missing anything?

**Comment by pookmish** [comment]
which part of the screenshots are you not getting? are you sure your browser caches are cleared and you are on the correct branch?

**Comment by jenbreese** [comment]
This is what mine looks like. I'm on the right branch. 
image

**Review by jenbreese — APPROVED** [review]
This works for me. thank you!

**Pull request merged**